doc/Command_Index_User.tex

```diff
@@ -978,6 +978,14 @@ \subsubsection{Hyphenation}
   & \texttt{zero} & The hyphen is considered to take up no space
 \end{argtable}
 
+\macroname{\textbackslash gresethyphenprotrusion}{\{\#1\}}{gregoriotex-spaces.tex}
+Sets the protrusion factor of hyphens at end of lines. The default value is 50, a good compromise in the tested scores.
+
+\begin{argtable}
+  \#1 & \texttt{number} & The protrusion factor, over 100
```

Comment:
I understand what this means, but it took me a few seconds to "get it".  I think you should say "The protrusion factor as a percentage" or "The protrusion factor, percent".